adafruit_displayio_sh1107.py

```diff
@@ -65,23 +66,45 @@
 
 
 # Sequence from sh1107 framebuf driver formatted for displayio init
-_INIT_SEQUENCE = (
-    b"\xae\x00"  # display off, sleep mode
-    b"\xdc\x01\x00"  # display start line = 0 (POR = 0)
-    b"\x81\x01\x2f"  # contrast setting = 0x2f
-    b"\x21\x00"  # vertical (column) addressing mode (POR=0x20)
-    b"\xa0\x00"  # segment remap = 1 (POR=0, down rotation)
-    b"\xcf\x00"  # common output scan direction = 15 (0 to n-1 (POR=0))
-    b"\xa8\x01\x7f"  # multiplex ratio = 128 (POR)
-    b"\xd3\x01\x60"  # set display offset mode = 0x60
-    b"\xd5\x01\x51"  # divide ratio/oscillator: divide by 2, fOsc (POR)
-    b"\xd9\x01\x22"  # pre-charge/dis-charge period mode: 2 DCLKs/2 DCLKs (POR)
-    b"\xdb\x01\x35"  # VCOM deselect level = 0.770 (POR)
-    b"\xb0\x00"  # set page address = 0 (POR)
-    b"\xa4\x00"  # entire display off, retain RAM, normal status (POR)
-    b"\xa6\x00"  # normal (not reversed) display
-    b"\xaf\x00"  # DISPLAY_ON
-)
+# we fixed sh110x addressing in 7, so we have slightly different setups
+if sys.implementation.version[0] < 7:
```

Comment:
hmmmm .. it seems like this lib is also used on standard python3 (is in pypi) where sys.implementation.version is currently 3.  I guess this means that python3 also keeps using the old behavior, which is fine for now.

Reply:
i dont even think that oleds are supported in blinka? or this special oled method

Reply:
I don't know, but it's in pypi. https://pypi.org/project/adafruit-circuitpython-displayio-sh1107/

I think it's fine, since 3<7 so this doesn't change anything on blinka.